ehitchhiking-ui/src/app/shared/components/active-trips-modal/active-trips-modal.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="modal-wrapper trips-modal-container">
+  <app-pre-loading [isLoading]="loading" [size]="loaderSize"></app-pre-loading>
+  <header class="modal-header form-row">
+    <span mat-dialog-title class="trips-title">Active trips</span>
+    <i class="material-icons icon-size">
+      history
+    </i>
+  </header>
+  <mat-expansion-panel>
+    <mat-expansion-panel-header>Filters</mat-expansion-panel-header>
+    <div class="expansion-panel-sm">
+      <mat-form-field>
+        <mat-label>Role</mat-label>
+        <mat-select [(ngModel)]="selectedRole" name="selected">
+          <mat-option (click)='filterByRole()' *ngFor="let role of roles" [value]="role.value">
+            {{role.viewValue}}
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+    </div>
+  </mat-expansion-panel>
+  <div mat-dialog-content class="trips">
+
+    <div class="trips-container" *ngFor="let trip of tripsArray|
+    filterByRole:role.roleField:selectedRole:role.isEnable|
+     slice: 0:limit; trackBy: trackById">
+      <app-active-trip *ngIf="!isShowTripInfo" class="trip" [trip]="trip"
+                       (click)="showTripInfo(trip)"></app-active-trip>
+    </div>
+    <div class="scroll-marker" #sMarker></div>
+    <app-active-trip-additional-info [trip]="tripInfo"   *ngIf="isShowTripInfo"
+                                     ></app-active-trip-additional-info>
+    <!--    <app-no-data *ngIf="!tripsArray.length" isEmpty="true"></app-no-data>-->
+  </div>
+
+  <div class="modal-close-control">
```

Comment:
As far as I remember, modal close control should be in the header. Pls look at other modals

Reply:
in other modals also